openproblems/tasks/dimensionality_reduction/methods/phate.py

```diff
@@ -2,31 +2,45 @@
 from ....tools.normalize import log_cpm_hvg
 from ....tools.normalize import sqrt_cpm
 from ....tools.utils import check_version
+from typing import Optional
 
 
-def _phate(adata, test=False, n_pca=None):
+def _phate(adata, test: bool = False, n_pca: Optional[int] = None, gamma: float = 1):
     from phate import PHATE
 
     if test:
         n_pca = n_pca or 10
     else:  # pragma: no cover
         n_pca = n_pca or 100
 
-    phate_op = PHATE(n_pca=n_pca, verbose=False, n_jobs=-1)
+    phate_op = PHATE(n_pca=n_pca, verbose=False, n_jobs=-1, gamma=gamma)
     adata.obsm["X_emb"] = phate_op.fit_transform(adata.X)
+    adata.uns["method_code_version"] = check_version("phate")
```

Comment:
Does this change to how versions are handled need to be made to all the methods?

Reply:
It does, eventually. https://github.com/openproblems-bio/openproblems/pull/463 handles this.